pyracf/common/security_admin.py

```diff
@@ -222,6 +222,10 @@ def _to_steps(self, results: Union[List[dict], dict, bytes]) -> Union[dict, byte
         steps_dictionary = {}
         for step, result_dictionary in enumerate(pre_processed_results):
             steps_dictionary[f"step{step+1}"] = result_dictionary
+        self.__logger.log_dictionary(
+            "Steps Dictionary",
+            steps_dictionary,
+        )
```

Comment:
I don't know if displaying the security result steps dictionary is necessary. All security result dictionary are already displayed in the logs before this. I am not sure if this adds much and if we do keep it, there needs to be a conditional that checks if `self.__debug` is `True` before attempting to log the dictionary.

Reply:
That is a totally fair point, I can scrap this. I was thinking we would need it for unit testing, but you are right that all the steps were already displayed in the log.